Sources/Testing/Running/XCTestScaffold.swift

```diff
@@ -167,7 +167,20 @@ public enum XCTestScaffold: Sendable {
   /// ## See Also
   ///
   /// - <doc:TemporaryGettingStarted>
+#if SWIFT_PM_SUPPORTS_SWIFT_TESTING
+  @available(*, deprecated, message: "This version of Swift Package Manager supports running swift-testing tests directly.")
+#endif
   public static func runAllTests(hostedBy testCase: XCTestCase) async {
+#if SWIFT_PM_SUPPORTS_SWIFT_TESTING
+    let message = warning("This version of Swift Package Manager supports running swift-testing tests directly.", options: .forStandardError)
```

Comment:
Can the message also state that because SwiftPM has support built-in, `XCTestScaffold` itself will _not_ run tests? Just to clarify that SwiftPM will be taking precedence and explain the effect of this warning.

Also, less important but maybe something along the lines of "Usage of `XCTestScaffold` can safely be removed."?

Reply:
If you have specific phrasing, I'll take it.

I am leaving out the "remove this" part for now because this is going to be fragile between toolchain versions. I'll add it when we get to a point where we can assume everybody has a recent-enough toolchain.

Reply:
```suggestion
    let message = warning("Ignoring XCTestScaffold, since this version of Swift Package Manager runs tests directly.", options: .forStandardError)
```

One thing I like about this phrasing is that it not only says SwiftPM _supports_ running tests, but actually confirms that it _will_ run them (or perhaps already has, by this point). In other words it avoids confusion about whether the user needs to adopt something to enable the new SwiftPM behavior; they get it for free.

Reply:
How about:

> This version of Swift Package Manager supports running swift-testing tests directly. Ignoring XCTestScaffold.runTests(hostedBy:).